solr/modules/monitor/src/java/org/apache/solr/monitor/MonitorConstants.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one or more
+ *  * contributor license agreements.  See the NOTICE file distributed with
+ *  * this work for additional information regarding copyright ownership.
+ *  * The ASF licenses this file to You under the Apache License, Version 2.0
+ *  * (the "License"); you may not use this file except in compliance with
+ *  * the License.  You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package org.apache.solr.monitor;
+
+import org.apache.lucene.monitor.QueryDecomposer;
+
+public class MonitorConstants {
+
+  private MonitorConstants() {}
+
+  public static final String REVERSE_SEARCH_PARAM_NAME = "reverseSearch";
+  public static final String SOLR_MONITOR_CACHE_NAME = "solrMonitorCache";
```

Comment:
Added https://github.com/apache/solr/pull/2382/commits/e12ec6fa880c67b31d2078c5f410ed517e98f530 commit which proposes to make the cache name configurable.

Reply:
looks good!